app/routes/application/review.js

```diff
@@ -50,7 +50,22 @@ module.exports = router => {
           }
         }
       }
-    }
+
+      pageObject.errorList.push({
+        text: 'You cannot apply to ‘Gorse SCITT – Primary (X100)’ because it has no vacancies',
```

Comment:
Gorse SCITT - Prumary (X100) no longer has vacancies — choose another course

Reply:
Not necessarily.